internal/mode/static/nginx/runtime/manager.go

```diff
@@ -18,7 +17,6 @@ import (
 const (
 	pidFile            = "/var/run/nginx/nginx.pid"
 	pidFileTimeout     = 10000 * time.Millisecond
```

Comment:
Does it make sense to have the timeout apply to the entire reload operation, including checking for the pid file?

Reply:
Got it. Makes sense!